contrib/devtools/update-translations.py

```diff
@@ -150,6 +155,10 @@ def postprocess_translations(reduce_diff_hacks=False):
                 numerus = message.get('numerus') == 'yes'
                 source = message.find('source').text
                 translation_node = message.find('translation')
+
+                if contains_bitcoin_addr(translation_node.text):
+                    print('Warning! Address found in translation in file %s. This will be removed' % (filepath))
+                    translation_node.clear()
```

Comment:
Why not move this with the `check_format_specifiers` check below. It means you can re-use the !valid code.
```
                     if not valid: # set type to unfinished and clear string if invalid
                         translation_node.clear()
                         translation_node.set('type', 'unfinished')
have_errors = True
```

Reply:
Good call. I have updated this. I'm not all that familiar with Python so if there's a better way of formatting what I have now, let me know.